### ISSUE TITLE
CSP failure prevents login

### ISSUE THREAD

**Issue by markspolakovs** [Alex-Rose/fb-messenger-cli#83]
When trying to log in, I get the following output:

```
Facebook credentials
Email: [REDACTED]
Password: ************************
Warning: Errors caught in return data
/home/marks/.nvm/versions/node/v6.11.1/lib/node_modules/fb-messenger-cli/scripts/login.js:34
      objData.saveTime = new Date().getTime();
                       ^

TypeError: Cannot set property 'saveTime' of undefined
    at Socket.<anonymous> (/home/marks/.nvm/versions/node/v6.11.1/lib/node_modules/fb-messenger-cli/scripts/login.js:34:24)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

I added some debugging statements (specifically, `console.log(phantom.data)` before line 25 in `scripts/login.js`, and this is what the script gets instead of JSON:

```
Refused to load the font 'https://static.xx.fbcdn.net/rsrc.php/yH/r/jJ5jE8KIrbS.ttf' because it violates the following Content Security Policy directive: "font-src *.messenger.com data:".

Refused to load the font 'https://static.xx.fbcdn.net/rsrc.php/yl/r/3b5d06LC2bM.svg' because it violates the following Content Security Policy directive: "font-src *.messenger.com data:".

Refused to get unsafe header "X-FB-Debug"
Blocked a frame with origin "file://" from accessing a frame with origin "null".  The frame requesting access has a protocol of "file", the frame being accessed has a protocol of "about". Protocols must match.
```

**Versions:**

Node: 6.11.1

npm: 3.10.10

fb-messenger-cli: 1.6.3

phantomjs-prebuilt: 2.1.14 (part of the base package, but may be relevant)

OS: Ubuntu 17.04

**Comment by SamBergeron** [comment]
Hey if you want to pull the latest code and let me know if you still see this thanks @markspolakovs

**Comment by SamBergeron** [comment]
Also fixed as part of 4f1faa2701e8d5442332a1388488fb13036c30f1
